docs/integrations/sources/convex.md

```diff
@@ -0,0 +1,79 @@
+# Convex
+
+This page contains the setup guide and reference information for the Convex source connector.
+
+Get started with Convex at the [Convex website](https://convex.dev).
+See your data on the [Convex dashboard](https://dashboard.convex.dev/).
+
+## Overview
+
+The Convex connector supports Full Refresh, Incremental Append, and Incremental Dedupe with deletes.
+
+### Output schema
+
+This source syncs each Convex table as a separate stream.
+Check out the list of your tables on the [Convex dashboard](https://dashboard.convex.dev/) in the "Data" view.
+
+Types not directly supported by JSON are encoded as described in the
+[JSONSchema](https://json-schema.org/understanding-json-schema/reference/index.html)
+for the stream.
+
+For example, the Javascript value `new Set(["a", "b"])` is encoded as `{"$set": ["a", "b"]}`, as described by the JSONSchema
+`{"type": "object", "description": "Set", "properties": {"$set": {"type": "array", "items": {"type": "string"}}}}`.
```

Comment:
looking at https://docs.airbyte.com/understanding-airbyte/supported-data-types#the-types and `shape_enum_to_json_schema`

could we map `Value::Int64` to `integer` and/or `big_integer`?

Reply:
Not if we continue using our default json encoding, which does base64 on ints. I've been putting off creating a more consumable json encoding because I don't want to maintain two yet. I figured we can wait until we're sure we need it and know what it should look like.

Reply:
sounds good. let's add a "format" parameter to the API then? we can have it start as "convex-json" or something like that and then add new ones over time.

(come to think of it, we should do this for our other endpoints too...)

Reply:
yep sounds good. should i do that before sending the PR to airbyte?